### PR TITLE
p1 b1 r13 paris

### PR DESCRIPTION
**Pull request by GrantSchreib**

### CODE REVIEW

File: bHY6yh-de3608/cod-bxxqqq_bHY6yh-de3608.xml [hunk 1]
```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title>Rubrica 13 [Krakow, lat 700 III Transcription]</title>
```
Comment (jeffreycwitt) [inline bHY6yh-de3608/cod-bxxqqq_bHY6yh-de3608.xml:5]: can you remove the [Krakow...] stmt
```suggestion
            <title>Rubrica 13</title>
```

File: bHY6yh-de3608/cod-bxxqqq_bHY6yh-de3608.xml [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+            <author ref="#per-c7y65a">Geremia da Montagnone</author>
+            <respStmt>
+               <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+            <respStmt>
+               <name>Katrina Vander Ahe</name>
```
Comment (jeffreycwitt) [inline bHY6yh-de3608/cod-bxxqqq_bHY6yh-de3608.xml:13]: You should also put your name here.

```suggestion
               <name>Grant</name>
```

File: bHY6yh-de3608/cod-bxxqqq_bHY6yh-de3608.xml [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 13 [Krakow, lat 700 III Transcription]</title>
```
Comment (jeffreycwitt) [inline bHY6yh-de3608/cod-bxxqqq_bHY6yh-de3608.xml:20]: The Krakow statement should be removed here as well.

```suggestion
               <title>Rubrica 13</title>
```

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]
Everything looks pretty good. Just two requested changes in the header.

**Comment by GrantSchreib** [comment]
corrections made

**Review by jeffreycwitt — APPROVED** [review]
Looks good.

**Pull request merged**